main/src/main/scala/sbt/Defaults.scala

```diff
@@ -2885,9 +2885,9 @@ object Classpaths {
                         excl: FileFilter): Classpath =
     (base * (filter -- excl) +++ (base / config.name).descendantsExcept(filter, excl)).classpath
 
-  def autoPlugins(report: UpdateReport, internalPluginClasspath: Seq[File]): Seq[String] = {
+  def autoPlugins(report: UpdateReport, internalPluginClasspath: Seq[File], scalaVersion: String): Seq[String] = {
```

Comment:
This will fail MiMa (which sbt-validator doesn't check).

Reply:
Thanks @dwijnand , what do you suggest to do here? It seems it is no way to call `scalaVersion.value` directly in the method.

Reply:
I'm thinking method overloading. Existing callees won't benefit from the change, but indirect users will.

Reply:
Thanks a lot, it's a good idea!